tests/test_parse.py

```diff
@@ -12,9 +12,9 @@ def test_main():
     path = "result.csv"
     main(path)
 
-    with open(CORRECT_QUOTES_CSV_PATH, "r") as correct_file, open(
-        path, "r"
-    ) as result_file:
+    with open(
+        CORRECT_QUOTES_CSV_PATH, "r", encoding="utf-8"
+    ) as correct_file, open(path, "r", encoding="utf-8") as result_file:
```

Comment:
please donʼt change tests

Reply:
windows is using different default encoding so i had to change tests for the code to work on my machine. turns out it's not an issue for the github environment. would be nice to have a hint about that but now i know